CONTRIBUTING.md

```diff
@@ -21,7 +21,7 @@ accept, pull requests within a few days. We may suggest some changes or improvem
 
 Some things that will increase the chance that your pull request is accepted:
 
-* Use CoffeeScript [idioms](http://arcturo.github.io/library/coffeescript/04_idioms.html) and [style guide](https://github.com/polarmobile/coffeescript-style-guide)
```

Comment:
This would be a good place to mention we're using [standard](https://github.com/feross/standard) for linting.